src/ui/public/elasticsearch_errors/is_term_size_zero_error.js

```diff
@@ -1,6 +1,10 @@
 import ElasticsearchError from './elasticsearch_error';
 
 export default function isTermSizeZeroError(error) {
-  const esError = new ElasticsearchError(error);
-  return esError.hasRootCause('size must be positive, got 0');
+  try {
+    const esError = new ElasticsearchError(error);
+    return esError.hasRootCause('size must be positive, got 0');
+  } catch (err) {
+    return false;
+  }
```

Comment:
This kind of solution would need to be repeated in any other error services we may add to `elasticsearch_errors`, which makes it seem like a workaround for a design mistake I made in `ElasticsearchError`. I think that class should just not throw an error at all. What do you think?